config/environments/development.rb

```diff
@@ -78,4 +78,10 @@
     api_enabled: true,
     migration_enabled: true,
   }
+
+  # Disable origin check for Cross-Site Request Forgery (CSRF) protection for codespaces.
```

Comment:
Does this happen when the CodeSpace is running in a browser or locally via VS Code? Or both? 🤔

Reply:
I tested it running in a browser as I figured that's how the content editors are going to use it... its an odd one but there's a few similar reports kicking around in various GitHub issues about it

Reply:
Yeah, I've seen it in the past but from memory the process is smoother when running from VS Code locally. But the fix  here looks like it'll do the trick nicely 👌🏽